dev.levz.TinyImageFinder.yml

```diff
@@ -0,0 +1,18 @@
+app-id: dev.levz.TinyImageFinder
+runtime: org.kde.Platform
+runtime-version: '5.15'
+sdk: org.kde.Sdk
+command: image-finder
+finish-args:
+  - --share=ipc
+  - --socket=x11
+  - --socket=wayland
```

Comment:
if it supports waylandm then line 8 should be `fallback-x11` instead.. otherwise remove the wayland socket.

Reply:
Fixed to fallback-x11.